prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/administrators-page/administrators-page.component.html

```diff
@@ -1,13 +1,13 @@
 <app-page [busy]="busy">
-  <app-page-header>PharmaNet Site Registration</app-page-header>
+  <app-page-header>PharmaNet Health Authority Information</app-page-header>
```

Comment:
Apparently we don't want to see the words "Site Registration" anywhere in Health Authorities.